Pseudo/css/pseudoElements.css

```diff
@@ -0,0 +1,101 @@
+/*FirstLetter*/
+
+.firstLetter p::first-letter {
+	color: red;
+	font-size: 23px;
+}
+
+/*firstLine*/
+
+.firstLine p::first-line {
+	color: green;
+}
+
+/*selection*/
+
+.selection p::selection {
+	background-color: yellow;
+}
+
+/*beforeafterStyling*/
+
+.beforeafterStyling {
+	text-align: center;
+	overflow-x: hidden;
+}
+.beforeafterStyling h3{
+	text-align: left;
+}
+.beforeafterStyling h4 {
+	display: inline-block;
+	position: relative;
+}
+.beforeafterStyling h4::before,.beforeafterStyling h4::after {
+	position: absolute;
+	content: '';
+	width: 900px;
+	height: 2px;
+	background-color: red;
+	top: 10px;
+}
+.beforeafterStyling h4::before {
+	right: 100%;
+	margin-right: 15px;
+}
+.beforeafterStyling h4::after {
+	left: 100%;
```

Comment:
use right: 0 instead

Reply:
Why?

Reply:
I am not getting the logic behind why we need to shift an element 100% from the left? Also I pulled the assignment and figured out that the logic behind beforeAfterStyling is not the most simple or standard way to do it. The logic needs rework. `<h4>` must be display: block, not inline-block;

Reply:
I m not getting  your solution push it by doing in the safe file after this.